app/views/arrangement/view_payment_plan.scala.html

```diff
@@ -124,7 +124,7 @@ <h2 class="govuk-heading-m">@messages("ssttp.arrangement.print-summary.schedule.
     </dl>
 
     <p class="govuk-body">
-        <a id="print-summary-link" href="" class="govuk-link" class="js-visible trackedPrintLink">
+        <a id="print-summary-link" href="" class="govuk-link js-visible" class="js-visible trackedPrintLink">
```

Comment:
isn't this newc `js-visible` redundant? There was already one there

Reply:
Indeed